src/components/structures/CustomRoomGroupFilterPanel.js

```diff
@@ -22,7 +22,7 @@ import dis from '../../dispatcher/dispatcher';
 import classNames from 'classnames';
 import * as FormattingUtils from '../../utils/FormattingUtils';
 
-class CustomRoomTagPanel extends React.Component {
+class CustomRoomGroupFilterPanel extends React.Component {
```

Comment:
The `CustomRoomTagPanel` was already correctly named - it should not be modified.